### PR TITLE
text-transform and text-decoration with formattedText fixed

### PR DESCRIPTION
**Pull request by enchev**
Fix for #1280

### CODE REVIEW

File: utils/utils.ios.ts [hunk 1]
```diff
@@ -49,108 +49,103 @@ export module ios {
...
+
+            for (let i = 0; i < v.formattedText.spans.length; i++) {
+                let span = v.formattedText.spans.getItem(i);
+                span.text = getTransformedText(v, span.text, transform);
+                console.log("HERE: " + span.text);
+            }
         } else {
-            setTextDecorationNative(view, view.text || view.attributedText, NSMutableDictionary.alloc().init());
+            let source = v.text;
+            let attributes = NSMutableArray.array();
```
Comment (hshristov) [inline utils/utils.ios.ts]: According to ios-runtime it will be much faster if we pass JS array here.

File: utils/utils.ios.ts [hunk 2]
```diff
@@ -49,108 +49,103 @@ export module ios {
...
+            let range = { location: 0, length: source.length };
+
+            var decorationValues = (decoration + "").split(" ");
+
+            if (decorationValues.indexOf(enums.TextDecoration.none) === -1) {
+                let dict = NSMutableDictionary.new();
```
Comment (hshristov) [inline utils/utils.ios.ts]: Same change for Map<> here.

**Comment by hshristov** [comment]
:+1: but it would be better if we switch NSArray & NSDictionary to JS Array and JS Map.

**Comment by dtopuzov** [comment]
run ci

**Pull request merged**